universe/supplycommit/mock.go

```diff
@@ -75,9 +75,9 @@ type fundPsbtMockFn func(
 	chainfee.SatPerKWeight, int32,
 ) (*tapsend.FundedPsbt, error)
 
-// signAndFinalizePsbtMockFn defines a type for the mock function used in
+// signPsbtMockFn defines a type for the mock function used in
 // SignAndFinalizePsbt, to simplify a long type assertion.
-type signAndFinalizePsbtMockFn func(
+type signPsbtMockFn func(
```

Comment:
nit (pre-existing): Can we also please fix the folding/indentation here? Looks like the return params fit on the same line as the input params.